sql/account_balance.sql

```diff
@@ -0,0 +1,23 @@
+-- liquibase formatted sql
+
+-- changeset devwasabi2:create-debit-account-transactions-total-view
+CREATE VIEW debit_account_total AS
+SELECT 
+    debit_account_id,
+    SUM(transaction_amount)::NUMERIC(26, 3) AS total_amount
+FROM 
+    transaction
+GROUP BY 
+    debit_account_id;
+-- rollback DROP VIEW IF EXISTS debit_account_total;
+
+-- changeset devwasabi2:create-credit-account-transactions-total-view
+CREATE VIEW credit_account_total AS 
+SELECT 
+    credit_account_id,
+    SUM(transaction_amount)::NUMERIC(26, 3) AS total_amount
+FROM 
+    transaction
+GROUP BY 
+    credit_account_id;
+-- rollback DROP VIEW IF EXISTS credit_account_total;
```

Comment:
Change this file views.sql

Reply:
yea, I forgot to change from numeric to bigint here as well